cmd/keeper/keeper.go

```diff
@@ -93,6 +95,8 @@ func init() {
 	cmdKeeper.PersistentFlags().StringVar(&cfg.pgConfDir, "pg-conf-dir", "", "absolute path to user provided postgres configuration. If empty a default dir under $dataDir/postgres/conf.d will be used")
 	cmdKeeper.PersistentFlags().StringVar(&cfg.pgSUUsername, "pg-su-username", "", "postgres superuser user name (required by pg_rewind)")
 	cmdKeeper.PersistentFlags().StringVar(&cfg.pgSUPassword, "pg-su-password", "", "postgres superuser password (required by pg_rewind)")
+	cmdKeeper.PersistentFlags().StringVar(&cfg.initialSUUsername, "initial-su-username", "", "postgres initial superuser username")
+	cmdKeeper.PersistentFlags().StringVar(&cfg.initialSUPasswordFile, "initial-su-password-secret-file", "", "postgres initial superuser password secret file")
```

Comment:
I also wanted to read it from a file when adding `pgSUPassword` for pg rewind instead of relying of an argument or env variable. I'll create a future PR the make it consistent with your proposal.